src/map/utils/battleutils.cpp

```diff
@@ -4363,6 +4362,62 @@ namespace battleutils
         return reductionPercent;
     }
 
+    void BindBreakCheck(CBattleEntity* PAttacker, CBattleEntity* PDefender)
+    {
+        if (PDefender->StatusEffectContainer->HasStatusEffect(EFFECT_BIND))
+        {
+            uint16 BindBreakChance = 0; // 0-1000 (100.0%) scale. Maybe change to a float later..
+            uint16 LvDiffByExp = charutils::GetRealExp(PAttacker->GetMLevel(), PDefender->GetMLevel()); // This is temp.
+
+            // Todo: replace with an actual calculated value based on level difference. Not it, Bro!
+            // This entire block of conditionals should not exist. Lv diff really shouldn't be handled at the exp check level either.
+            // It might not even be in sync with the check values the entire way from lv 1 to lv 99 for all we know.
+            if ((PAttacker->objtype == TYPE_PC || PAttacker->objtype == TYPE_PET) && PDefender->objtype == TYPE_MOB)

```

Comment:
the the type check is because I don't want to call those GetRealExp function for non players. as for the rate on players, if you have a formula you'd like me to use, pls share